src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -1329,9 +1394,15 @@ def _get_child_names(self):
                         singleton_names.append(member)
                 creatable_type_names = struct_field.creatabletypes
                 command_names = [x.name for x in struct_field.commands]
+                if hasattr(struct_field, "queries"):
+                    query_names = [x.name for x in struct_field.queries]
+        if hasattr(struct_field, "queries"):
+            return singleton_names, creatable_type_names, command_names, query_names
         return singleton_names, creatable_type_names, command_names
```

Comment:
I think it is fine to return an empty list for query_names in this case, the check at lines 1404, 1415 won't be needed

Reply:
Done. Thank you.